### PR TITLE
element-{web,desktop}: 1.8.2 -> 1.8.4

### PR DESCRIPTION
**Pull request by Ma27**
###### Motivation for this change

https://github.com/vector-im/element-desktop/releases/tag/v1.8.4
https://github.com/vector-im/element-web/releases/tag/v1.8.4

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/instant-messengers/element/element-desktop-package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "element-desktop",
   "productName": "Element",
   "main": "lib/electron-main.js",
-  "version": "1.8.2",
+  "version": "1.8.4",
```
Comment (Ma27) [inline pkgs/applications/networking/instant-messengers/element/element-desktop-package.json:5]: It's fairly weird that nothing else in here has changed. Investigating.

Comment (Ma27) [reply pkgs/applications/networking/instant-messengers/element/element-desktop-package.json:5]: OK, doesn't matter here, matrix-js-sdk is directly pulled from `pkgs.element-web` where I just confirmed that it got updated.

**Review by sumnerevans — APPROVED** [review]
Builds and runs on my machine

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 137645` at c81983ec run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>element-desktop</li>
    <li>element-desktop-wayland</li>
  </ul>
</details>
<details>
  <summary>3 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/build-tools-in-build-inputs.md">build-tools-in-build-inputs</a>

yarn is a build tool so it likely goes to `nativeBuildInputs`, not `buildInputs`.

Near pkgs/development/tools/yarn2nix-moretea/yarn2nix/default.nix:295:7:
```
    |
295 |       buildInputs = [ yarn nodejs rsync ] ++ extraBuildInputs;
    |       ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/build-tools-in-build-inputs.md">build-tools-in-build-inputs</a>

rsync is a build tool so it likely goes to `nativeBuildInputs`, not `buildInputs`.

Near pkgs/development/tools/yarn2nix-moretea/yarn2nix/default.nix:295:7:
```
    |
295 |       buildInputs = [ yarn nodejs rsync ] ++ extraBuildInputs;
    |       ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/development/tools/yarn2nix-moretea/yarn2nix/default.nix:335:7:
```
    |
335 |       installPhase = attrs.installPhase or ''
    |       ^
```
</li>
  </ul>
</details>

**Review by djmaze — APPROVED** [review]

**Pull request merged**